docs/src/pages/guides/angular.md

```diff
@@ -24,7 +24,7 @@ module.exports = {
 };
 ```
 
-Checkout the [orval config](../reference/orval-config) reference to see all available options.
+Checkout the [orval config](../reference/configuration/full-example) reference to see all available options.
```

Comment:
```suggestion
Checkout the [orval configuration](../reference/configuration/overview) reference to see all available options.
```

Reply:
The link `[orval config](../reference/configuration/full-example)` is used in this place in al of the below:
https://github.com/anymaniax/orval/blob/master/docs/src/pages/guides/react-query.md
https://github.com/anymaniax/orval/blob/master/docs/src/pages/guides/swr.md
https://github.com/anymaniax/orval/blob/master/docs/src/pages/guides/svelte-query.md
https://github.com/anymaniax/orval/blob/master/docs/src/pages/guides/vue-query.md

Are you sure you want it to be different in case of Angular?

Reply:
True